### PR TITLE
Add medical_dialogue, clean up demo.

### PR DESCRIPTION
**Pull request by nsthorat**
- Remove spacy_ner from demo datasets
- Add medical_dialog
- Add the_movies_dataset (fix issues with public bucket.. we should probably make a kaggle source)

### CODE REVIEW

File: demo.yml [hunk 1]
```diff
@@ -55,34 +55,33 @@ datasets:
...
+  - name: databricks-dolly-15k-curated-en
+    namespace: lilac
+    settings:
+      ui:
+        media_paths:
+          - original-instruction
+          - original-context
+          - original-response
+          - - new-instruction
+            - value
```
Comment (dsmilkov) [inline demo.yml:67]: i think you can make this an array, like in line 80

File: scripts/deploy_demo.py [hunk 2]
```diff
@@ -68,11 +68,6 @@ def deploy_demo(overwrite: bool, skip_sync: bool, skip_load: bool, skip_build: b
   if not skip_load:
     load(DEMO_DATA_DIR, DEMO_CONFIG_PATH, overwrite)
 
-  # Copy lilac concepts to the demo data dir from the default data_path: 'data'.
```
Comment (dsmilkov) [inline scripts/deploy_demo.py:71]: is this gone because deploy_hf does it?

**Review by dsmilkov — APPROVED** [review]

File: demo.yml (hunk 1 above)
Comment (nsthorat) [reply demo.yml:68]: done

File: scripts/deploy_demo.py (hunk 2 above)
Comment (nsthorat) [reply scripts/deploy_demo.py:71]: yeah, it happens when we push source code now

**Pull request merged**